public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
do we need more expressive names here?

Reply:
Replaced with scriba. It's the name of app, so maybe title will suit better than generic "React app".